website/docs/user-guides/uninstall-litmus.md

```diff
@@ -20,15 +20,18 @@ To disconnect the [ChaosAgent](../getting-started/resources#chaosagents) connect
 To remove the respective components of the ChaosAgents you need to manually delete the created resources of that ChaosAgent.
 
 ```bash
-kubectl delete chaosengine <CHAOSENGINE_IDs> --all -A
-kubectl delete chaosexperiments <CHAOSEXPERIMENTS_IDs> --all -A
-kubectl delete chaosresults <CHAOSRESULTS_IDs> --all -A
+kubectl delete chaosengine <CHAOSENGINE_IDs> --all -<AGENT_NAMESPACE>
+kubectl delete chaosexperiments <CHAOSEXPERIMENTS_IDs> --all -<AGENT_NAMESPACE>
+kubectl delete chaosresults <CHAOSRESULTS_IDs> --all -<AGENT_NAMESPACE>
+kubectl delete workflows <WORKFLOW_IDs> --all -<AGENT_NAMESPACE>
+kubectl delete deployment chaos-operator-ce chaos-exporter --all -<AGENT_NAMESPACE>
 ```
 
 To remove all the ChaosAgents component ever created from the system, apply this command.
 
 ```bash
-kubectl delete chaosengine,chaosexperiments,chaosresults --all -A
+kubectl delete chaosengine,chaosexperiments,chaosresults,workflows --all -A
+kubectl delete deployment chaos-operator-ce chaos-exporter --all -A
```

Comment:
If we are removing the infra components separately, it may be good to delete the service account,  role and role bindings too. Also the chaos engine, chaos experiment, chaos result CRD (if they are not deleted during agent disconnection) although they are not transient resources.

Reply:
cc: @ajeshbaby @ksatchit @rajdas98

Reply:
Have added steps to delete sa, cluster role bindings and cluster role for both scopes